src/Message/PurchaseRequest.php

```diff
@@ -70,11 +70,14 @@ private function sign($params, $api_key) {
 
 	public function sendData($data)
     {
+		// FIXME: You're not actually sending any data here.
```

Comment:
I'm trying to help @sanderha with the gateway implementation. I was wondering why you'd have to send data there? I guess the data would need to be sent if this would be using the REST API, but currently it's just a redirect to an external payment form.
Other gateways I looked at (such as `GoCardless` also seem to not send any data there). I've also used the same pattern in for a Gateway I implemented: https://github.com/bummzack/omnipay-postfinance/blob/master/src/Message/PurchaseRequest.php

I'm thankful for suggestions if you think this approach is flawed.